src/components/buttons/ExportButton.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { BiExport } from 'react-icons/bi';
+
+import { DataSourceContext } from '@/hooks';
+
+export interface ImportButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  className?: string;
+}
+
+export default function ImportButton({
+  className = 'btn btn-primary',
+  ...props
+}: ImportButtonProps): JSX.Element {
+  const { isLoaded, datasource } = React.useContext(DataSourceContext);
+  const ref = React.useRef<HTMLAnchorElement>(null);
+
+  return (
+    <>
+      <button
+        id="export-button"
+        type="button"
+        disabled={!isLoaded}
+        className={className}
+        onClick={() => {
+          if (ref.current) {
+            const rawBook = datasource?.sqljsManager.exportDatabase() as Uint8Array;
+            const blob = new Blob([rawBook], { type: 'application/vnd.sqlite3' });
```

Comment:
In the future we can use `pako.gzip` to compress the data but given Maffin supports importing both compressed and uncompressed data, doesn't make a difference and, this way, users can check the file in sqlite directly.